clusters/build-clusters/common_except_app.ci/icsp.yaml

```diff
@@ -5,5 +5,5 @@ metadata:
 spec:
   repositoryDigestMirrors:
   - mirrors:
-    - registry.svc.ci.openshift.org/ocp/4.4-art-latest
+    - registry.ci.openshift.org/ocp/4.4-art-latest
```

Comment:
Don't we need a similiar line for all 4.X versions?

Reply:
That is the question in my PR summary. :)

I did not do it right away because i would like e.g.,

```
    - registry.ci.openshift.org/ocp/4.5-art-latest
    source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
```

or

```
    - registry.ci.openshift.org/ocp/4.4-art-latest
    - registry.ci.openshift.org/ocp/4.5-art-latest
    source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
```

How does it work?

quay.io/openshift-release-dev/ocp-v4.0-art-dev is backed up by different branches?

Reply:
I don't know either

Reply:
@smarterclayton  needs to weigh in. I'm not actually sure why there's anything on these past the registry DN... also ICSP only applies to images referred to by SHA so is not going to make much of a difference in our workflows - I can't remember how this was supposed to work for us last year.

Reply:
This set up registry on api.ci as a preferred cache for images that could also be on quay.  If quay went down, we'd have the images already.  This is a search path for digest in general, so it's a fallabck list - the more fallbacks, the slower the performance.  So you want the mirror deployed on a cluster to roughly be in a list with the most likely place the image exists, followed by the second most likely, down to the fallback quay.

This could fairly easily be removed and only introduced if quay went down for a sustained period.  Now that quay has the read only fallback that SRE runs, I think most of the need for this has passed, and we should simply remove this ICSP config completely.